EIPS/eip-4494.md

```diff
@@ -1,30 +1,30 @@
 ---
 eip: 4494
-title: Permit for ERC-721 NFTs
-description: ERC-712-singed approvals for ERC-721 NFTs
+title: Permit for EIP-721 NFTs
+description: ERC-712-singed approvals for EIP-721 NFTs
```

Comment:
@Pandapip1 it seems a false positive warning for EIPW? My understanding is that preamble can't add Markdown Links.